siriuspy/siriuspy/search/id_search.py

```diff
@@ -160,6 +160,12 @@ class IDSearch:
             'cv2': 'SI-10SB:PS-CV-2:Current-SP',
             'qs1': 'SI-10SB:PS-QS-1:Current-SP',
             'qs2': 'SI-10SB:PS-QS-2:Current-SP',
+            'qd1_1': 'SI-10M1:PS-QDB1:Current-SP',
```

Comment:
the indices here are confusing to me.... I expect the last index would be the enumeration, not the first one...

Reply:
I agree, @anacso17 . But since we have ch1/cv1/qs1, ch2/cv2/qs2 to refer to upstream correctors (numerical index right after prefixes ch, cv and qs), the other indices dor first and second normal quadrupoles at each side of de ID have to be somewhere...

what is your proposed set of names for 

(ch1, ch2)
(cv1, cv2)
(qs1, qs2)
(qd1_1, qd2_1)
(qf1, qf2)
(qd1_2, qd2_2)

?

Reply:
what about:
(ch_1, ch_2)
(cv_1, cv_2)
(qs_1, qs_2)
(qd1_1, qd1_2)
(qf_1, qf_2)
(qd2_1, qd2_2)
?

Reply:
yes, I liked this.